checks/system/ce/ce_import_run_image.py

```diff
@@ -91,17 +91,14 @@ def set_image_path(self):
     @run_before('run')
     def set_executable_opts(self):
         self.executable_opts = [
-            '-benchmark', '-fp64',
-            f'-numbodies={self.num_bodies_per_gpu * self.num_gpus_per_node}',
-            f'-numdevices={self.num_gpus_per_node}'
+            f'-d0', f'-n{self.stream_array_size}',
```

Comment:
not critical, but I would move @run_after('setup') = line 86 up before  @run_before('run') = line 78